### PR TITLE
Stripe - `hero` cleanup

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/SDS-253

#### Description
Hides the text protection scrim when no `backgroundImage` is passed in. Removes support for `hero` prop in the `Stripe` component

### CODE REVIEW

**Comment by mperrotti** [comment]
Doesn't break anything in `mup-web`, and `hero` doesn't even get used in pro-web. Review at will

File: assets/scss/components/_forms.scss [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+---
+Faux inputs visually mimic actual input elements.
+
+```html_example
+<div class="chunk">
+	<div class="fauxInput">I'm a faux input</div>
+</div>
+```
+*/
+.fauxInput {
```
Comment (akdetrick) [inline assets/scss/components/_forms.scss]: can we move `fauxInput` to the partial for the component in which it's used?

File: assets/scss/components/_forms.scss [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+	<input id="check2" class="checkbox" value="blue" type="checkbox" />
+	Blue
+</label>
+<label for="check3" class="label--minor chunk">
+	<input id="check3" class="checkbox" value="blue" type="checkbox" checked />
+	Already checked
+</label>
+```
+*/
+.checkbox {
```
Comment (akdetrick) [inline assets/scss/components/_forms.scss]: let's move this to a `_checkbox` partial to match the component.

File: src/layout/Stripe.jsx [hunk 3]
```diff
@@ -44,11 +44,11 @@ class Stripe extends React.Component {
 				className={classNames}
 				style={styles}
 				{...other}>
-				{ hero ?
+				{ backgroundImage ?
 					(
-						<Bounds className='stripe-heroContent'>
+						<div className='stripe-aboveScrim'>
```
Comment (akdetrick) [inline src/layout/Stripe.jsx]: do we need to rethink the `heroContent` prop for modals with this change?

Comment (mperrotti) [inline src/layout/Stripe.jsx]: I don't think so. "Hero" is a pretty understood term, I didn't even intentionally name it the same thing as the `<Stripe>` class.
If you don't think this is intuitive, I'm happy to rethink

File: assets/scss/components/_forms.scss (hunk 1 above)
Comment (mperrotti) [inline assets/scss/components/_forms.scss]: `fauxInput` is only used on `NumberInput` right now, but it could be used for similar things in the future

**Comment by mperrotti** [comment]
bump

File: src/layout/Stripe.jsx [hunk 4]
```diff
@@ -21,7 +21,7 @@ class Stripe extends React.Component {
 			backgroundImage,
 			collection,
 			inverted,
-			hero,
+			// hero,
```
Comment (mmcgahan) [inline src/layout/Stripe.jsx]: remove commented line?

**Pull request merged**